src/main/java/org/dcsa/ebl/controller/ShippingInstructionsSummariesTOController.java

```diff
@@ -20,16 +20,16 @@
 @RestController
 @RequiredArgsConstructor
 @RequestMapping(value = "shipping-instructions-summaries", produces = {MediaType.APPLICATION_JSON_VALUE})
-public class ShippingInstructionsSummariesTOController extends AbstractTOController<ShippingInstructionTOService> {
+public class ShippingInstructionsSummariesTOController extends AbstractTOController<ShippingInstructionService> {
 
     private final ExtendedParameters extendedParameters;
 
-    private final ShippingInstructionTOService shippingInstructionTOService;
+    private final ShippingInstructionService shippingInstructionTOService;
 
     private final R2dbcDialect r2dbcDialect;
```

Comment:
is this still needed? @nt-as

Reply:
It is used directly in the class, so it is still necessary.

We can probably remove it (hide it in a base case) when we are ready to bump to the slimmed core framework.